example/urls.py

```diff
@@ -1,13 +1,12 @@
-from django.conf.urls import url
+from django.urls import re_path
 from rest_framework import routers
 
 from example import views
 
-
 router = routers.SimpleRouter()
 router.register(r'post', views.PostViewSet)
 
 urlpatterns = router.urls + [
-    url(r'^post-view/(?P<pk>[0-9]+)', views.approved_function_view),
-    url(r'^post-cbv/(?P<pk>[0-9]+)', views.ApprovedClassBasedView.as_view()),
+    re_path(r'^post-view/(?P<pk>[0-9]+)', views.approved_function_view),
+    re_path(r'^post-cbv/(?P<pk>[0-9]+)', views.ApprovedClassBasedView.as_view()),
```

Comment:
Do we need the regular expression or would the following also do it? I personally find this more pleasing to the eye but I'll leave it up to you.
```
    path('post-view/<int:pk>', views.approved_function_view),
    path('post-cbv/<int:pk>', views.ApprovedClassBasedView.as_view()),
```